src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -4027,26 +4040,21 @@ private DynamicMetaObject CannotIndexTarget(DynamicMetaObject target, DynamicMet
                 bindingRestrictions);
         }
 
-        internal static bool CanIndexFromEndWithNegativeIndex(DynamicMetaObject target)
+        internal static bool CanIndexFromEndWithNegativeIndex(
+            DynamicMetaObject target,
```

Comment:
Turns out this parameter is not used within the method.

I think we probably should keep the existing checks for the common scenarios, since `Module.ResolveMethod` is relatively expensive.